backend/src/controllers/memberController.ts

```diff
@@ -50,10 +56,13 @@ const updateMember = async (req: Request, res: Response) => {
 
 //delete members
 const deleteMembers = async (req: Request, res: Response) => {
+  /* 
+  #swagger.tags = ['Members']
+  */
   const { id } = req.params;
 
   try {
-    const member = await prisma.member.delete({
+    const member = await prisma.member.deleteMany({
```

Comment:
Why this change
And even if it's an optimization or a correction, it has nothing to do with swagger and open API.
So it shouldn't be in this PR

Reply:
This change is about swagger.
swagger assumes anything in the pattern .delete() to be api endpoints.
So prisma.member.delete() would be considered to be an api endpoint which is not the case.